ui/pages/_document.js

```diff
@@ -37,7 +37,7 @@ class MesheryDocument extends Document {
       <Html lang="en" dir="ltr">
         <Head>
           <meta charSet="utf-8" />
-          <link rel="icon" href="/static/favicon.ico" />
+          <link rel="icon" href="/static/favicon.png" />
```

Comment:
This will change the favicon in all the pages, right?

Reply:
Yess, Ig it changes in all the pages. I had run the app after my changes and it was successful